components/optimizer/greedymax.py

```diff
@@ -0,0 +1,59 @@
+from .base import BaseOptimizer
+from typing import Dict
+from common.minimodel.group import Group
+from common.minimodel.plan import Plan
+from components.selector import GroupInfo
+
+Selection = Dict[Group, GroupInfo]
```

Comment:
I think we should get rid of this since now we have an actual defined `Selection` in `common.calculations`.

Reply:
That greedymax.py file is not being used and probably be replaced with Bryan's work

Reply:
Ah... okay. I mistakenly thought it was a blueprint to guide him. Let's make sure he knows that this file is outdated.